components/adv-post-form.js

```diff
@@ -93,16 +91,16 @@ export default function AdvPostForm ({ children }) {
               )
             }}
           </VariableInput>
-          {me && router.query.type === 'discussion' &&
+          {me &&
             <Checkbox
               label={
                 <div className='d-flex align-items-center'>crosspost to nostr
                   <Info>
                     <ul className='fw-bold'>
-                      <li>crosspost this discussion item to nostr</li>
+                      <li>crosspost this item to nostr</li>
```

Comment:
Hmm yeah I agree, since this message sits in it's own modal as well I don't think it will feel crammed.

How about something like this:
<img width="937" alt="image" src="https://github.com/stackernews/stacker.news/assets/53542748/7f9a73ac-2e54-4050-8f9c-d893fd3d2a01">

Reply:
Mhh, I would make it context aware. Also, would use `NIP-01` instead of `nip01` to be consistent with the other lines that already mention NIPs. So if you're writing a discussion post, it would show

> crosspost this discussion as a NIP-23 event

or while writing a link

> crosspost this link as a NIP-01 event

etc.

_(btw, polls use NIP-41 events, not NIP-44)_

Reply:
Ahh good idea, this is what I have now.
https://github.com/stackernews/stacker.news/blob/fc8518050b6086f734caa6066f2b064cae16492a/components/adv-post-form.js

First line is dynamic and describes the nip being used for the item type:
```
function renderCrosspostDetails(itemType) {
    switch (itemType) {
      case 'discussion':
        return <li>crosspost this discussion as a NIP-23 event</li>;
      case 'link':
        return <li>crosspost this link as a NIP-01 event</li>;
      case 'bounty':
        return <li>crosspost this bounty as a NIP-99 event</li>;
      case 'poll':
        return <li>crosspost this poll as a NIP-41 event</li>;
      default:
        return null;
    }
  }
```

The rest is static:
```
{me && itemType &&
            <Checkbox
              label={
                <div className='d-flex align-items-center'>crosspost to nostr
                  <Info>
                    <ul className='fw-bold'>
                      {renderCrosspostDetails(itemType)}
                      <li>requires NIP-07 extension for signing</li>
                      <li>we use your NIP-05 relays if set</li>
                      <li>we use these relays by default:</li>
                      <ul>
                        {DEFAULT_CROSSPOSTING_RELAYS.map((relay, i) => (
                          <li key={i}>{relay}</li>
                        ))}
                      </ul>
                    </ul>
                  </Info>
                </div>
            }
              name='crosspost'
            />}
```